Source/AlertController.swift

```diff
@@ -110,6 +110,10 @@ public final class AlertController: UIViewController {
     /// on an action. If it returns false, the AlertAction handler will not be executed.
     @objc
     public var shouldDismissHandler: ((AlertAction?) -> Bool)?
+    
+    /// A closure called before the alert is dismissed
+    @objc
+    public var willBeDismissedHandler: (() -> Void)?
```

Comment:
Can you name this `willDismissHandler`?

Reply:
Also, please clarify in the comment that this only works if the alert is dismissed through its own dismiss method, not if that’s done manually.

Reply:
Yes, of course. Thanks for your review!